src/lib/seam/thermostats/use-update-fan-mode.ts

```diff
@@ -36,7 +36,7 @@ export function useUpdateFanMode(): UseMutationResult<
     },
     onSuccess: (_data, variables) => {
       queryClient.setQueryData<ThermostatDevice | null>(
-        ['thermostats', 'get', { device_id: variables.device_id }],
+        ['devices', 'get', { device_id: variables.device_id }],
```

Comment:
Nice catch. Any other ones with this issue?

Reply:
Yes, I decided to do this in a separate PR for the other hook: #562